public/components/custom_panels/helpers/utils.tsx

```diff
@@ -468,6 +468,11 @@ export const displayVisualization = (metaData: any, data: any, type: string) =>
   if (metaData === undefined || isEmpty(metaData)) {
     return <></>;
   }
+
+  if (metaData.user_configs !== undefined) {
```

Comment:
would it ever be empty string?

Reply:
I don't believe so, this corrects the formatting error not being readable from Log Events -> Panels

Reply:
Can you please elaborate on this. Didn't quite get the explanation here.

Reply:
In Panel the saved object has the availability config but the visual is unable to parse it correctly so it treats it as undefined and fails to display it.

Panel View
<img width="1089" alt="Screenshot 2023-07-07 at 10 13 49 AM" src="https://github.com/opensearch-project/dashboards-observability/assets/105462877/ae493341-f718-40a6-9cba-37bb677d8d4a">
Log Event - Visualization
<img width="1259" alt="Screenshot 2023-07-07 at 10 14 35 AM" src="https://github.com/opensearch-project/dashboards-observability/assets/105462877/b93a48e3-73fb-411c-b45c-9ceff9103a81">